WORKSPACE

```diff
@@ -70,12 +70,10 @@ http_archive(
 
 http_archive(
     name = "org_tensorflow",
-    patch_args = ["-p1"],
-    patches = ["//third_party/tensorflow:tf.patch"],
```

Comment:
The master branch is synchronized with tf-nightly, which requires this patch.  To build against tf 2.13, use the 2.13 branch.  That one seems to build fine with tf 2.13 installed.